sklearn/compose/tests/test_column_transformer.py

```diff
@@ -310,6 +310,50 @@ def test_column_transformer_empty_columns(pandas, column_selection,
     assert isinstance(ct.transformers_[0][1], TransRaise)
 
 
+def test_column_transformer_transformers_output():
+    X_array = np.array([[0, 1, 2], [2, 4, 6]]).T
```

Comment:
I missed the `.T` and thought there were 3 features: let's just write it 3x2 then please

Reply:
This line just follows the pattern of other tests (see [here](https://github.com/scikit-learn/scikit-learn/blob/bcc35eeacbf5ea1f96b4745e26db86b5d759d98b/sklearn/compose/tests/test_column_transformer.py#L77) and [here](https://github.com/scikit-learn/scikit-learn/blob/bcc35eeacbf5ea1f96b4745e26db86b5d759d98b/sklearn/compose/tests/test_column_transformer.py#L140), for example). Is it worth creating an inconsistency?

Reply:
Let's just explicitly set the shape then, since we don't care about what's in the array. Maybe just do `np.arange(6).reshape(3, 2)`?